### PR TITLE
[Registrar] Academic calculator styling.

### LINKED ISSUE #7933: GPA Calculator branding and styling
# Story

As a webteam, we would like to include some additional theming of the GPA calculator so that it uses University Branding and looks nice.

# Additional Information

https://registrar.prod.drupal.uiowa.edu/gpa-calculator - what is there now is still a deployment behind as of 8/20th.

We did minimal theming previously since this is a contrib module. We thought we likely should add some University Branding to this and otherwise clean up its appearance for display on the Registrar site.
```[tasklist]
### Tasks
- [x] Add new CSS file
- [x] Add new `registrar_core.libraries.yml` file to add an library item for the CSS file
```

### PR DESCRIPTION
**Pull request by GaryRidgway**
Closes https://github.com/uiowa/uiowa/issues/7933

## To test

1. Sync the registrar site
2. go to https://registrar.uiowa.ddev.site/gpa-calculator
3. Mess with the calculator. ensure it still works!
4. Do a code review. Is there something that I goofed?

### CODE REVIEW

**Comment by joewhitsitt** [comment]
pulling to test

File: docroot/sites/registrar.uiowa.edu/modules/registrar_core/registrar_core.module [hunk 1]
```diff
@@ -14,5 +16,30 @@ function registrar_core_preprocess_block(&$variables) {
...
+    $form['gpa_row']['actions']['add_row']['#attributes']['class'][] = 'button--full-width';
+    $form['#submit'][] = 'registrar_core_post_submit';
   }
 }
+
+/**
+ * Triggers after submit to change the way that the grade output is generated.
+ */
+function registrar_core_post_submit(&$form, FormStateInterface $form_state) {
+  $form['#attached']['library'][] = 'uids_base/callout';
```
Comment (joewhitsitt) [inline docroot/sites/registrar.uiowa.edu/modules/registrar_core/registrar_core.module]: Does this need to trigger each submit or can it just load as part of the form alter?

**Comment by joewhitsitt** [comment]
Screenshot 2024-08-26 at 8 57 34 AM

Should mobile give a bit more room to the grade dropdown and less to the credits earned? Once selected I can't see what grade it is.

Tested with iphone se emulator in chrome/safari

File: docroot/sites/registrar.uiowa.edu/modules/registrar_core/registrar_core.module (hunk 1 above)
Comment (GaryRidgway) [reply docroot/sites/registrar.uiowa.edu/modules/registrar_core/registrar_core.module]: We might be able to load on the form alter, but i remember there being some odd things in conjunction with the block regenerating. I can test it in the form alter and get back to this.

**Comment by GaryRidgway** [comment]
> Screenshot 2024-08-26 at 8 57 34 AM
> Should mobile give a bit more room to the grade dropdown and less to the credits earned? Once selected I can't see what grade it is.
> 
> Tested with iphone se emulator in chrome/safari

Yes, I will try and see what I can do about the size.

File: docroot/sites/registrar.uiowa.edu/modules/registrar_core/registrar_core.module (hunk 1 above)
Comment (GaryRidgway) [reply docroot/sites/registrar.uiowa.edu/modules/registrar_core/registrar_core.module]: I have added this! Thanks for the catch Joe :D

**Review by joewhitsitt — APPROVED** [review]
Thanks for the tweaks. Looks good.

**Pull request merged**